### ISSUE TITLE
Black window version 7.2.0

### ISSUE THREAD

**Issue by mackignacio** [a85/POSTMan-Chrome-Extension#6726]
**Describe the bug**
Postman shows only a black window after updating to version 7.2.0. I already did the suggestion on #4703 but that doesn't work. I also try to install older version of postman but it keeps on updating to version 7.2.0.

**To Reproduce**
1. Install any version of Postman.
2. Update it to 7.2.0
3. Run postman wait for 10 to 20 second after the initialization then screen will get blacked out.

**Expected behavior**
Want to see the default window working properly.

**Screenshots**
![image](https://user-images.githubusercontent.com/12345233/59606822-a98e7500-9144-11e9-843f-56cb7099eeb8.png)

**App information (please complete the following information):**
 - Native App
 - Postman Version [7.2.0](https://dl.pstmn.io/download/version/7.2.0/windows64)
 - OS: Windows 10 x64

**Additional context**
The app is working fine on version 7.1.1 but after the update it also not working even if you install it back to version 7.1.1.

**Comment by fillx** [comment]
I have a same problem. Postman 7.2.0. Windows 8.1 pro x64.

**Comment by harryi3t** [comment]
@Mackignacio Did you try setting an environment variable `POSTMAN_DISABLE_GPU=true` as mentioned in the other thread https://github.com/postmanlabs/postman-app-support/issues/4703? You can verify that in a terminal by running `echo %POSTMAN_DISABLE_GPU%`, you should see the output as `true` (case sensitive)

Can you also try a custom canary build that we had released a few days back here https://github.com/postmanlabs/postman-app-support/issues/4594#issuecomment-497646826

**Comment by mackignacio** [comment]
@harryi3t  Yes I did and I already mentioned it on my issue. I already fix it by uninstalling Postman then deleting all the postman folder on my %appdata% folder. Both on local and roaming folder I also deleted those postman folder. Then I reinstall the newest version 7.2.0 then it run smoothly.

I wonder why when you update it will no longer work. Maybe the problem is on the automatic update by electron.
